### PR TITLE
DT-3033 Promote to Prod

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

**Review by xoen — APPROVED** [review]

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: incentives-api.hmpps.service.justice.gov.uk
+    contextColour: green
+
+  env:
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
```
Comment (petergphillips) [inline helm_deploy/values-prod.yaml:12]: shouldn't need this variable - it is the default in the `values.yaml` anyway

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: incentives-api.hmpps.service.justice.gov.uk
+    contextColour: green
```
Comment (petergphillips) [inline helm_deploy/values-prod.yaml:9]: would have been better putting `contextColour` in values.yaml and then you wouldn't need to set here

File: helm_deploy/values-prod.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
```
Comment (petergphillips) [inline helm_deploy/values-prod.yaml:5]: same as the default value anyway so not necessary

Comment (Mjwillis) [reply helm_deploy/values-prod.yaml:5]: 👍

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (Mjwillis) [reply helm_deploy/values-prod.yaml:12]: 👍

**Pull request merged**